kudos-services/src/test/java/org/exoplatform/kudos/test/service/KudosServiceTest.java

```diff
@@ -528,4 +529,33 @@ public void onEvent(Event<KudosService, Kudos> event) throws Exception {
     assertTrue(listenerInvoked.get());
   }
 
+  @Test
+  public void testGetKudosByActivityId() throws Exception {
+    KudosService kudosService = getService(KudosService.class);
+    KudosStorage kudosStorage = getService(KudosStorage.class);
+    Kudos kudos = newKudosDTO();
+    kudos.setEntityType(KudosEntityType.USER_PROFILE.name());
+    kudos=kudosService.createKudos(kudos, SENDER_REMOTE_ID);
+    Kudos storedKudos=kudosStorage.getKudoById(kudos.getTechnicalId());
+    entitiesToClean.add(kudos);
+    KudosEntity newKudos = kudosService.getKudosByActivityId(storedKudos.getActivityId());
+    assertNotNull(newKudos);
```

Comment:
Can you add more assertion please. You can use this method [compareResults](https://github.com/Meeds-io/kudos/blob/develop/kudos-services/src/test/java/org/exoplatform/kudos/test/BaseKudosTest.java#L157)